### ISSUE TITLE
Config.unobserve is deprecated.

### ISSUE THREAD

**Issue by Stanzilla** [AtomLinter/linter-php#22]
Config::unobserve no longer does anything. Call `.dispose()` on the object returned by Config::observe instead.
```
Config.unobserve (C:\Users\Benjamin\AppData\Local\atom\app-0.196.0\resources\app.asar\src\config.js:1060:19)
LinterPhp.destroy (C:\Users\Benjamin\.atom\packages\linter-php\lib\linter-php.coffee:29:16)
LinterView.remove (C:\Users\Benjamin\.atom\packages\linter\lib\linter-view.coffee:252:6)
<unknown> (C:\Users\Benjamin\.atom\packages\linter\lib\linter-view.coffee:124:7)
```

**Comment by joshbadger** [comment]
+1

**Comment by kevinsawicki** [comment]
Fixed by #29  and published in `0.0.13`
